cfpb/cfgov-refresh
# Failsafe for Ask CFPB landing-page answer slugs

Our template for the Ask CFPB landing page used a wagtail template
function for finding a page by slug, but it was using the original
slug creation value instead of the current page's slug.
This PR gets the URL directly from the page, which will always be
current.

## Testing locally
- Try the current link for the first answer under **Student loans** at  [/ask-cfpb/](http://localhost:8000/ask-cfpb/) 
- Edit the English 'question' value for [answer 545](http://localhost:8000/admin/ask_cfpb/answer/edit/545/) and save, then publish its [English page](http://localhost:8000/admin/pages/8412/). The landing page link for the first answer under **Student loans** should still work even though the question text (and thus the Answer object's slug) has changed.
